CMakeLists.txt

```diff
@@ -10,8 +10,23 @@
 cmake_minimum_required(VERSION 3.5 FATAL_ERROR)
 project(valhalla LANGUAGES CXX C)
 
+include(FindPkgConfig)
+include(GNUInstallDirs)
+
 list(INSERT CMAKE_MODULE_PATH 0 ${CMAKE_SOURCE_DIR}/cmake)
 
+option(ENABLE_CCACHE "Speed up incremental rebuilds via ccache" ON)
+option(ENABLE_HTTP "Enable HTTP" ON)
+option(ENABLE_DATA_TOOLS "Enable Valhalla data tools" ON)
+option(ENABLE_SERVICES "Enable Valhalla services" ON)
+option(ENABLE_PYTHON_BINDINGS "Enable Python bindings" OFF)
```

Comment:
This should probably be turned back on. When we build packages we want to by default build all the things.